src/actions/app.js

```diff
@@ -124,6 +125,9 @@ export const saveNew = org => (dispatch, getState) => {
         dispatch(addNotification('Codesplaination Saved!'));
         // Clear the hasUnsavedChanges flag
         dispatch(saveSucceeded());
+        // Update the user's personal and organization snippet lists
+        dispatch(updateUserSnippets());
+        dispatch(fetchSnippetLists());
```

Comment:
`fetchSnippetLists` will fetch the snippets for the user and the organizations he/she belongs to, so the call to `updateUserSnippets` is unnecessary